examples/goroutines/goroutines.go

```diff
@@ -1,4 +1,5 @@
-// A _goroutine_ is a lightweight thread of execution.
+// Una _goroutine_ è di fatto un [green thread](https://en.wikipedia.org/wiki/Green_threads) che permette di parallelizzare
```

Comment:
http://blog.golang.org/concurrency-is-not-parallelism ;)

"rendere concorrente"?